client/src/components/ScrollMenu.js

```diff
@@ -27,7 +26,7 @@ const Image = styled.img`
 `;
 
 export const ScrollMenu = () => {
-  const { data, loading, error, doFetch } = useAsync(() => getData(limited));
+  const { data, loading, error, doFetch } = useAsync(() => getData());
```

Comment:
You can directly bypass `getData` instead of a new function which calls `getData`.

```suggestion
  const { data, loading, error, doFetch } = useAsync(getData);
```